### PR TITLE
Pond tests

### PR DESCRIPTION
**Pull request by ankita0805**
- Pond test in write followed by read mode
- Pond test in double buffer mode
- Parameterize read start_addr for write/read test

### CODE REVIEW

**Review by kavyasreedhar — APPROVED** [review]
looks good to me

File: tests/test_pond_rd_wr.py [hunk 1]
```diff
@@ -39,42 +39,43 @@ def test_pond(data_width=16,  # CGRA Params
                                   check_flip_flop_always_ff=False)
 
     tester = fault.Tester(magma_dut, magma_dut.clk)
-    ###
-    # Ranges, Strides, Dimensionality, Starting Addr, Starting Addr - Schedule
-    ctrl_rd = [[16, 1], [1, 1], 2, 0, 16]
-    ctrl_wr = [[16, 1], [1, 1], 2, 0, 0]
-    pond_config = generate_pond_api(ctrl_rd, ctrl_wr)
+    # Parameterize on start addr for read
```
Comment (mbstrange2) [inline tests/test_pond_rd_wr.py:42]: Can you use the pytest parameterize to do this?

Here's an example - https://github.com/StanfordAHA/lake/blob/master/tests/test_reg_fifo.py#L12

**Review by mbstrange2 — CHANGES_REQUESTED** [review]
Just refactor to use purest param and I'll approve

**Comment by ankita0805** [comment]
Obsolete.

**Pull request closed without merging**